components/patient-measurements/api/src/main/java/org/phenotips/measurements/internal/AbstractMeasurementHandler.java

```diff
@@ -166,6 +199,51 @@ public void initialize() throws InitializationException
         this.chartConfigurations = this.settingsFactory.loadConfigurationsForMeasurementType(getName());
     }
 
+    @Override
+    public Collection<VocabularyTerm> getAssociatedTerms(Double standardDeviation)
+    {
+        ResourceBundle configuration = ResourceBundle.getBundle("measurementsAssociatedTerms");
+        List<VocabularyTerm> terms = new ArrayList<>();
+
+        if (null == standardDeviation || standardDeviation <= -3) {
+            addResolvedTermsToList(configuration, this.getName(), "extremeBelowNormal", terms);
+        }
+        if (null == standardDeviation || standardDeviation <= -2) {
```

Comment:
you don't need repeating null checks. It would have been caught by the first one. If you want, you can even initially do 
```java
if (standardDeviation == null) {
    standardDeviation = Double.NEGATIVE_INFINITY
}
```